hirosystems/stacks-blockchain-api/templates/api-reader/deployment.yaml

```diff
@@ -77,7 +77,7 @@ spec:
           {{- include "common.tplvalues.render" (dict "value" .Values.apiReader.initContainers "context" $) | nindent 8 }}
         {{- end }}
       containers:
-        - name: stacks-blockchain-api-reader
+        - name: stacks-blockchain-api
```

Comment:
I have no preference on this but I am curious why you decided to not rename this to `stacks-blockchain-api-ro`.

Reply:
I just didn't like when I'm searching through logs and switching between reader or writer pods that I also have to make sure the container name in the query is correct. It's less frustrating downstream if the containers are named the same.

I think it would make more sense to differentiate their names if both a reader and writer were running in the same pod.